### DISCUSSION TITLE
Grupo Teams

### DISCUSSION THREAD

**Discussion by xenyens** [Launch-X-Latam/MisionFrontEnd#435] [Team Posts]
Hola, estuve leyendo varios mensajes sobre el grupo de Teams, lo que yo hice fue hacer mi cuenta de Git, y responder el cuestionario [https://forms.office.com/r/Ddr5fxJbNk](url) de esa URL, te piden el enlace de tu cuenta a Git, responder dos preguntas más y el correo te llegará a Outlook, con la cuenta que nos dieron, allí viene el QR o Link que es la invitación a la clase de Teams.

**Comment by AlexiaMata95** [comment]
Muchas gracias Xeny!! ya pude :3

**Comment by plopez10** [comment]
![screenshot](https://user-images.githubusercontent.com/71240157/194615137-e2baa966-0029-4363-8dd1-da5fb2a25612.png)
Es lo que hice pero sigo en espera :)
Gracias

**Comment by xenyens** [comment]
Si, eso mismo me apareció, ahora a esperar para entrar. 🫶🏼

**Comment by bryansb** [comment]
Estamos en las mismas, jaja. Con todo, muchas gracias Xeny. :3

**Comment by DAVIDCAVALIERE** [comment]
Igual me aparece solicitud enviada, supongo que a esperar!

**Comment by gumodi35** [comment]
Hola! 👍 

Despues de haber completado el Onboarding 🔖, llenas el registro para unirte al grupo de Microsoft Teams, eso te dara un link para poder ingresar.</br>
Debes usar tu cuenta de Innovaccion para acceder.</br>
En ocasiones puede que entres a la organizacion inmediatamente, si no tienes que esperar un poco.</br>
Cuando tengas acceso a la organizacion el el grupo de Teams, por lo regular la primera vez no estaras en el grupo de Mision Frontend, solo queda esperar que te asignen el acceso.</br>
Luego de que te sea asignado el acceso, podras ver los link de las seciones y los diferentes canales.
Las seciones se quedan grabadas, por lo que las podras ver ondeman cuando te sea posible, se asigna un tiempo para ver las seciones, pero no te preocupes por eso.</br>
Si presentas algun problema, comunicate via instagram o via teams a contacto innovaccion.</br>
Cualquier otra duda puedes hacer plantearlas por este medio en la seccion de Q&A de la Organizacion.
Saludos! 💯

**Comment by carlosmmayo** [comment]
Gracias
